pkg/services/screenshot/option.go

```diff
@@ -41,3 +43,14 @@ func (s ScreenshotOptions) SetDefaults() ScreenshotOptions {
 	}
 	return s
 }
+
+//nolint:errcheck,gosec
```

Comment:
I am a little bit concerned about this.

probably the better approach would be `_, _ = h.Write...`?

Reply:
I did think that, but we've used nolint elsewhere for the same hash function. Will fix!

Reply:
ah. ok then